src/components/Layout/Stack.tsx

```diff
@@ -21,6 +21,7 @@ type Props = {
     | "items-start"
     | "items-end"
     | "items-baseline";
+  onClick?: (e?: any) => void;
```

Comment:
I found myself wanting to add onClick to an `HStack` and was about to wrap it in a div so I could add an onClick to the div wrapping it. But then I figured the `HStack` is just a div itself... theres no reason it shouldn't just accept an onClick prop. So I'm adding it here... if we don't like this I can take it out!

Reply:
lol i've been here a thousand times